plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
@umpirsky If you're interested, I've been doing something along those lines [here](https://github.com/mAAdhaTTah/wppb-mod).